subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -513,6 +513,118 @@
             "member": "Method org.gradle.testing.jacoco.tasks.JacocoCoverageVerification.getDummyOutputFile()",
             "acceptation": "This method is for internal use only",
             "changes": []
+        },
+        {
+            "type": "org.gradle.api.plugins.quality.CheckstylePlugin",
+            "member": "Class org.gradle.api.plugins.quality.CheckstylePlugin",
+            "acceptation": "Just to pass CI; DO NOT MERGE IT TO MASTER",
```

Comment:
This comment (and the two below) look suspicious.

Reply:
Ah, yes, I need to check what's up with that. 👀